tests/baselines/reference/tscWatch/programUpdates/config-file-is-deleted.js

```diff
@@ -84,9 +84,6 @@ Input::
 //// [/a/b/tsconfig.json] deleted
 
 Output::
->> Screen clear
-[[90m12:00:24 AM[0m] File change detected. Starting incremental compilation...
-
 [91merror[0m[90m TS5083: [0mCannot read file '/a/b/tsconfig.json'.
```

Comment:
It's hard to tell in the diff here, but after this message is printed, the process exits (see exitCode below), so it's sort of pointless if this message is printed or not here other than to maybe indicate that we noticed because of a file watcher update.

This same error is printed if we start a missing config file in the first place.